google-cloud-spanner/src/test/java/com/google/cloud/spanner/AsyncResultSetImplTest.java

```diff
@@ -195,7 +195,7 @@ public void withCallback() throws InterruptedException {
     finishedLatch.await();
     // There should be between 1 and 4 callbacks, depending on the timing of the threads.
     // Normally, there should be just 1 callback.
-    assertThat(callbackCounter.get()).isIn(Range.closed(1, 4));
+    assertThat(callbackCounter.get()).isIn(Range.closed(1, 5));
```

Comment:
Why does this value change?

Reply:
Tests pass without this change. Reverted this.